e2e/test/scenarios/native/ai-sql-fixer.cy.spec.ts

```diff
@@ -2,39 +2,44 @@ import { SAMPLE_DB_ID } from "e2e/support/cypress_data";
 
 const { H } = cy;
 
-const FIX_MESSAGE = "Fixes applied. Run your query to view results.";
-
-describe("scenarios > native > ai sql fixer", { tags: "@skip" }, () => {
+H.describeWithSnowplow("scenarios > native > ai sql fixer", () => {
   beforeEach(() => {
+    H.resetSnowplow();
     H.restore();
     cy.signInAsAdmin();
-    H.activateToken("pro-self-hosted");
+    H.enableTracking();
+    H.activateToken("bleeding-edge");
     cy.intercept("POST", "/api/ee/ai-sql-fixer/fix").as("fixSql");
   });
 
+  afterEach(() => {
+    H.expectNoBadSnowplowEvents();
+  });
+
   it("should be able to fix SQL with AI", () => {
     H.visitQuestionAdhoc({
       display: "table",
       dataset_query: {
         database: SAMPLE_DB_ID,
         type: "native",
         native: {
-          query: "SELECT1 * FROM ORDERS",
+          query: "SELECT1",
```

Comment:
Way simpler and easier on the CI (if we ever end up running the query). No need to fetch and render all orders.